README.md

```diff
@@ -137,3 +137,28 @@ In particular this means that this project may not depend on GPL-licensed or AGP
 as these would violate the terms of those libraries' licenses.
 
 The contents of this repository are protected by Crown Copyright (C).
+
+## Performance
+The performance of PS Adaptor was tested with JMeter tool.
+There was a series of tests run with the following setup and parameters:
+- PS Adaptor is run in ECS AWS environment with 4 CPUs and 16 GB memory.
+- For the message queue mq.m5.xlarge host type was used
+- RDS DB host type was set to db.t3.xlarge
```

Comment:
I think we should mention that the MHS adaptor was also being run, but that Spine responses were being simulated by JMeter.